pkg/apis/resource/types.go

```diff
@@ -963,22 +821,6 @@ type DeviceClassSpec struct {
 	// +optional
 	// +listType=atomic
 	Config []DeviceClassConfiguration
-
-	// Only nodes matching the selector will be considered by the scheduler
-	// when trying to find a Node that fits a Pod when that Pod uses
-	// a claim that has not been allocated yet *and* that claim
-	// gets allocated through a control plane controller. It is ignored
-	// when the claim does not use a control plane controller
-	// for allocation.
-	//
-	// Setting this field is optional. If unset, all Nodes are candidates.
-	//
-	// This is an alpha field and requires enabling the DRAControlPlaneController
-	// feature gate.
-	//
-	// +optional
-	// +featureGate=DRAControlPlaneController
-	SuitableNodes *core.NodeSelector
```

Comment:
And this field name is burned, unless we decide to bring it back for essentially the same purpose at some point, in some newer implementation of similar functionality?

Reply:
No object stored with v1beta1 as encoding will have that field, so we could reuse it by dropping support for v1alpha3.

Reply:
It might be worth adding tombstone comments, just in case.

Reply:
We need the tombstone comment for the protobuf field number. Otherwise one of the new features targeting 1.32 might reuse the number, something that we cannot allow yet in 1.32.